tools/db_stress.cc

```diff
@@ -3365,7 +3365,7 @@ class AtomicFlushStressTest : public StressTest {
       thread->stats.AddErrors(1);
     } else {
       size_t num = rand_column_families.size();
-      thread->stats.AddBytesForWrites(num, (sz + 1) * num);
+      thread->stats.AddBytesForWrites(static_cast<int>(num), (sz + 1) * num);
```

Comment:
I wonder if type of writes_/deletes_/range_deletions_ should be long or size_t
If we keep them as long why not static_cast to long?